services/hidconfigurator.md

```diff
@@ -0,0 +1,66 @@
+# HID Adapter Configurator
```

Comment:
I would just call this "HID Adapter", the fact that the service allows configuration is a side-effect

Reply:
and rename file to ``hidadapter``

Reply:
Done! Thanks @pelikhan